Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -171,6 +171,8 @@ open class NavigationMapView: UIView {
             show([activeRoute])
         case .all(_):
             show(routes)
+        case .custom(cameraOptions: _):
+            show(routes)
```

Comment:
I think these two cases can be merged into one? I think argument name in `.custom` one can be also skipped as it's not used anyway.

Reply:
Yes. Because the `RoutesPresentationStyle` is used in `navigationMapView.fitCamera(to:routesPresentationStyle:animated:)`, the custom routes or route options could be decided by the `routes` property instead. The `RoutesPresentationStyle` should works for custom `cameraOptions` only.